gulpfile.js

```diff
@@ -41,6 +41,7 @@ const testAppEnv = {
   TEST_APP_CONSENT_COOKIE_NAME_PREFIX: 'CONSENT_TEST_APP',
   TEST_APP_UPLOAD_LIABILITY_COOKIE_NAME: 'UPLOAD_LIABILITY_TEST_APP',
   TEST_APP_X_FRAME_OPTIONS: 'SAMEORIGIN',
+  TEST_APP_ADMIN_EMAIL_ADDRESS: 'educandu-test-admin@test.com',
```

Comment:
We should flag the JIRA ticket so that we remember to set and admin email in OMA 😬

Reply:
I will ask Ilka for an address and flag it, thanks (it's optional though, so it doesn't have to happen immediately).